cpp/src/converters/COOtoCSR.cuh

```diff
@@ -98,7 +98,15 @@ gdf_error ConvertCOOtoCSR(T* sources, T* destinations, int64_t nnz, CSR_Result<T
     CUDA_TRY(cudaMemcpy(&maxId2, maxId_it, sizeof(T), cudaMemcpyDefault));
     maxId = maxId > maxId2 ? maxId : maxId2;
     result.size = maxId + 1;
-
+    // Sending a warning rather than an error here as this may be intended and suported.
+    if (result.size > nnz ) {
+        std::cerr<< "WARNING: there are more vertices than edges in the graph ";
+        std::cerr<< ": V=" << result.size <<", E="<<nnz <<". ";
+        std::cerr<< "Sometime this is not intended and may cause performace and stability issues. ";
+        std::cerr<< "Vertex identifieres must be in the range 0-V where V is the number of vertices. ";
```

Comment:
0-V can be confusing (are 0 and V included in the range?), and I guess what you actually meant is 0, 1, ... , V-1.

[0, V) or 0, ... , V-1 might be clearer?

Reply:
I'd vote for [0, V), it's compact and precise - but I was trained as a mathematician.

Reply:
Yes, plus we already use [0, V) elsewhere in the doc. I'll fix that.

Reply:
done